app/src/test/java/org/dhis2/usescases/login/LoginPresenterTest.kt

```diff
@@ -239,13 +239,12 @@ class LoginPresenterTest {
     @Test
     fun `Should display message and hide fingerprint dialog when authenticate throws an error`() {
         whenever(
```

Comment:
should we change the test name? Is not hidingFingerPrintDialog anymore?

Reply:
Done!